kin/stellar/builder.py

```diff
@@ -68,3 +68,13 @@ def sign(self, secret=None):
         if not secret:  # only get the new sequence for my own account
             self.sequence = self.get_sequence()
         super(Builder, self).sign(secret)
+
+    def append_create_account_op(self, destination, starting_balance, source=None, trustline=False, issuer=None, asset_code=None):
```

Comment:
I'd pass in the asset object instead of breaking it into issuer and asset_code

Reply:
more over, instead of having multiple variables that are all tied to the same action (trustline, isser, asset_code) and that if one is defined all the rest have to be, you can change the API to:
```python
append_create_account_op(self, destination, starting_balance, source=None, pretrusted_asset: Union[Asset, None] = None):
```
have a single parameter: `pretrusted_asset` or `trust_asset`. The value will either be None, which will not pre trust. Or an Asset object that will trust that asset.